services/src/main/java/org/jboss/windup/web/services/model/RegisteredApplication.java

```diff
@@ -41,14 +43,14 @@
     private RegistrationType registrationType;
 
     @Column(length = 256)
-    @Size(min = 1, max = 256)
-    @NotNull
+    @NotBlankConstraint(message = "The application title must be set.")
+    @Size(min = 1, max = 256, message = "The application title must be set and not longer than 250 characters.")
     private String title;
 
     @Column(length = 2048)
-    @Size(min = 1, max = 2048)
-    @FileExistsConstraint
-    @NotNull
+    @NotBlankConstraint(message = "The application path must be set.")
+    @FileExistsConstraint(message = "The path does not exist on the server: ${validatedValue}")
+    @Size(min = 1, max = 2048, message = "The application title path must be set and not longer than 2048 characters.")
```

Comment:
Without i18n, I'm not sure that these changes are really improvements. I don't think that it was possible for it to be null before anyway due to the FileExistsConstraint. Anyway... not a big deal.